### DISCUSSION TITLE
browers_fallback_url for scene-viewer

### DISCUSSION THREAD

**Discussion by sprengerst** [google/model-viewer#2649] [Q&A]
### Description

Is there a  way to specify the browser_fallback_url for the scene-viewer intent? Especially if working with iFrames this could be really handy? Otherwise a user is redirected to the iFrames content page, which is not desired most of the time.

Best,
Stefan

### Version
- model-viewer: vX.X.X

### Browser Affected

- [ X] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [X ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [ ] WebXR
- [X ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
Not currently. It's probably possible, but we're generally moving away from Scene Viewer and toward WebXR for AR anyway as it has many advantages. Can you share why you prefer to stick with Scene Viewer over our default? We want to ensure WebXR is meeting our user's needs.

**Reply by sprengerst** [reply]
Hi, ok. We prefer sceneviewer in cases were we need a „take a picture“ functionallity.
